src/features/goblins/bank/components/WithdrawItems.tsx

```diff
@@ -163,6 +163,7 @@ export const WithdrawItems: React.FC<Props> = ({ onWithdraw }) => {
                 onClick={() => onAdd(itemName)}
                 image={details.image}
                 locked={locked}
+                canBeLongPressed={true}
```

Comment:
You can just pass with prop here as true is the default ie. `canBeLongPressed`

Reply:
Nice catch! Done.